src/Resources/config/processors.yaml

```diff
@@ -87,6 +87,12 @@ services:
         tags:
             - { name: !php/const Synolia\SyliusAkeneoPlugin\Processor\ProductAttribute\AkeneoAttributeProcessorInterface::TAG_ID }
 
+    Synolia\SyliusAkeneoPlugin\Processor\ProductAttribute\AssetAttributeProcessor:
+        arguments:
+            $akeneoAssetRepository: '@akeneo.repository.asset'
+        tags:
+            - { name: !php/const Synolia\SyliusAkeneoPlugin\Processor\ProductAttribute\AkeneoAttributeProcessorInterface::TAG_ID }
```

Comment:
Can we have a auto configure for the processors that implement the interface to get this tag automatically ?

Reply:
it will be done in another PR, I also must do it on all other processors etc...